wikipron.py

```diff
@@ -7,12 +7,15 @@
 import re
 import sys
 
-from typing import Callable, Iterator, Optional, TextIO, Tuple
+from typing import Callable, List, Iterator, Optional, TextIO, Tuple
 
 import iso639
 import requests
 import requests_html
 
+
+__version__ = "0.1.0"
```

Comment:
Sneaking in this change so that `wikipron.__version__` is available as in a real released package...

Reply:
Okay with me.